portiaui/app/helpers/has-job.js

```diff
@@ -0,0 +1,14 @@
+import Ember from 'ember';
+
+export default Ember.Helper.extend({
+    jobQ: Ember.inject.service(),
+
+    onJobCountChange: Ember.observer('jobQ.jobs.count', function() {
+        this.recompute();
+    }),
+
+    compute(params) {
+        const spider = params[0];
+        return this.get('jobQ.jobs').countForSpider(spider.get('id')) > 0;
```

Comment:
this can be called often. maybe it's better for the jobQ service to have a method that returns an Ember.Object with it's own computed properties just for that spider id